grizli/__init__.py

```diff
@@ -1,9 +1,7 @@
 """
 Pipeline for flexible modeling and extration of slitless spectroscopy
 """
-#__version__ = "0.1.1.dev"
-#from .version import __version__
-from .version import __version__, __long_version__, __version_hash__
+from .version import __version__
```

Comment:
`setuptools-scm` basically reads the latest tag and builds a version from that, writing it to `grizli/version.py` when you build (and by extension install) the package.

The latest tag is `1.6.0`.  So `setuptools-scm` will then create a `1.6.1.devXXX` version were XXX is the number of commits since the previous tag.  It will also attach a git hash and date to it.  So it looks like `1.6.1.dev101+g78d36ea.d20221026`.  That means that between tags, it's always a dev version of the next patch release.

If we decide to tag that commit as version `1.6.1`, then the version will be exactly that tag, and subsequent commits  will be versioned as `1.6.2.devXXX`, etc.